lib/libtlspool.c

```diff
@@ -857,12 +511,12 @@ int tlspool_starttls (int cryptfd, starttls_t *tlsdata,
 	regent.pfd = poolfd;
 	pthread_mutex_lock (&recvwait);		// Will await unlock by master
 	/* Determine the request ID */
-	if (registry_update (&entry_reqid, &regent) != 0) {
+	if (registry_update(&entry_reqid, &regent) != 0) {
 		closesocket(cryptfd);
 		errno = EBUSY;
 		return -1;
 	}
-	memset (&cmd, 0, sizeof (cmd));	/* Do not leak old stack info */
+	memset(&cmd, 0, sizeof(cmd));	/* Do not leak old stack info */
 	cmd.pio_reqid = entry_reqid;
```

Comment:
als we spaties gaan aanpassen krijgen we clashes -- dat moeten we ons echt afwennen -- bovendien: mijn code, mijn stijl, lijkt me prettiger het consistent te houden